tools/memory/g_var_profiler.sh

```diff
@@ -81,6 +81,10 @@ fi
 
 if [ -z ${ELF} ]; then
 	ELF="${MEMTOOL_PATH}/../../build/output/bin/tinyara"
+	if [ ! -f ${ELF} ]; then
+		echo "No valid ELF file"
```

Comment:
"No ELF file generated for this board" is this correct?

Reply:
Hm.. It's little bit different. It could be a case for no generated or could be a case for different file like realtek. Realtek has different file name, tinyara.axf. How about "${ELF} is not available"?

Reply:
yes looks fine..